frontend/package.json

```diff
@@ -23,22 +23,22 @@
     "@emotion/server": "^11.10.0",
     "@emotion/styled": "^11.10.6",
     "@material-tailwind/react": "^1.4.2",
-    "@mui/material": "^5.12.0",
+    "@mui/material": "^5.12.1",
     "@mui/utils": "^5.12.0",
     "@opentelemetry/api": "^1.4.1",
-    "@opentelemetry/exporter-metrics-otlp-proto": "^0.37.0",
-    "@opentelemetry/exporter-trace-otlp-proto": "^0.37.0",
-    "@opentelemetry/sdk-metrics": "^1.11.0",
-    "@opentelemetry/sdk-node": "^0.37.0",
-    "@tanstack/react-query": "^4.28.0",
+    "@opentelemetry/exporter-metrics-otlp-proto": "^0.38.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.38.0",
+    "@opentelemetry/sdk-metrics": "^1.12.0",
+    "@opentelemetry/sdk-node": "^0.38.0",
```

Comment:
I vaguely remember having some issues with v0.38 of opentelemetry when I was initially implementing it. I'll approve this PR but we may have to revert the OT version if stuff is broken.